subcommittees/mentoring/README.md

```diff
@@ -1,73 +1,86 @@
+	a discussion on [this etherpad][discussion]
+- support current mentoring activities or initiate new ones OR
+- join the mentoring committee
+	- Sign up for the mentoring committee [mailing list][comm-mailing-list] and attend the next mentoring committee meeting (scheduled on [this etherpad][meetings])
 
-We welcome new members to the subcommittee.  If you are interested in
-joining our community-building efforts, please email the [mailing
-list](#mailing-list).
+For a complete summary of different mentoring roles, see [this page](roles.md).  
 
-## Steering Comittee Liaison
+## Steering Committee Liaison
 
 * Rayna Harris (@raynamharris, [@raynamharris](https://twitter.com/raynamharris))
 
 ## Current Members
 
 * Belinda Weaver (@weaverbel, [@cloudaus](https://twitter.com/cloudaus))
-* Bill Mills (@BillMills, [@billdoesphysics](https://twitter.com/billdoesphysics))
-* Carol Willing (@willingc, [@WillingCarol](https://twitter.com/WillingCarol))
-* Christina Koch, Chair (@ChristinaLK, [@_christinaLK](https://twitter.com/_christinaLK))
-* Daniel Chen (@chendaniely, [@chendaniely](https://twitter.com/chendaniely))
 * Karin Lagesen (@karinlag, [@karinlag](https://twitter.com/karinlag))
 * Kate Hertweck (@k8hertweck, [k8hert](https://twitter.com/k8hert))
 * Marian Schmidt (@marschmi, [@micro_marian](https://twitter.com/micro_marian))
-* Mariela Perignon (@mperignon)
-* Michael Sarahan (@msarahan)
-* Phil Rosenfield (@philrosenfield, [@philrosenfield](https://twitter.com/philrosenfield))
+* Christina Koch (@ChristinaLK, [@_christinaLK](https://twitter.com/_christinaLK))
 * Rayna Harris (@raynamharris, [@raynamharris](https://twitter.com/raynamharris))
 * Susan McClatchy (@smcclatchy, [@SueMcclatchy](https://twitter.com/SueMcclatchy))
 * Tracy Teal (@tracykteal, [@tracykteal](https://twitter.com/tracykteal))
+* Erin Becker
+* Tobin Magle
+* Mark Laufersweiler
+* Mateusz Kuzak
 
 ## Past Members
 
 * Sheldon McKay (@mckays630)
 * Tiffany Timbers (@ttimbers, [@TiffanyTimbers](https://twitter.com/TiffanyTimbers))
+* Phil Rosenfield (@philrosenfield, [@philrosenfield](https://twitter.com/philrosenfield))
+* Bill Mills (@BillMills, [@billdoesphysics](https://twitter.com/billdoesphysics))
+* Carol Willing (@willingc, [@WillingCarol](https://twitter.com/WillingCarol))
+* Daniel Chen (@chendaniely, [@chendaniely](https://twitter.com/chendaniely))
+* Mariela Perignon (@mperignon)
+* Michael Sarahan (@msarahan)
 
-[blog]: https://software-carpentry.org/blog/
-[blog-archives]: https://software-carpentry.org/blog/categories/#SLUG
-[mailing-list]: http://lists.software-carpentry.org/listinfo/SLUG
-[mailing-list-archives]: http://lists.software-carpentry.org/pipermail/SLUG/
-[meetings]: http://pad.software-carpentry.org/scf-mentoring
+[discussion]: http://pad.software-carpentry.org/instructor-discussion
+[host-mailing-list]: https://groups.google.com/a/carpentries.org/forum/#!forum/discussion-hosts
```

Comment:
It sounded like using Google Groups was [still up in the air][1], but we can always land this and sort that out later.

[1]: http://lists.software-carpentry.org/pipermail/mentoring/2017-March/000782.html